commerce-account-service/src/main/java/com/liferay/commerce/account/service/impl/CommerceAccountGroupCommerceAccountRelLocalServiceImpl.java

```diff
@@ -54,6 +55,16 @@
 			externalReferenceCode = null;
 		}
 
+		CommerceAccountGroupCommerceAccountRel
+			commerceAccountGroupCommerceAccountRel2 =
+				commerceAccountGroupCommerceAccountRelPersistence.fetchByC_C(
+					commerceAccountGroupId, commerceAccountId);
+
+		if (commerceAccountGroupCommerceAccountRel2 != null) {
+			throw new DuplicateCommerceAccountGroupCommerceAccountRelException(
+				"Account already associated to AccountGroup ");
```

Comment:
Message should be `"Account already associated with AccountGroup"`, with no space at the end